### PR TITLE
Web UI: Sort group memberships of profile

### LINKED ISSUE #1198: Group Memberships on profile page - should be sorted alphabetically

### PR DESCRIPTION
**Pull request by czzrr**
Fixes #1198 

Screenshot from 2023-02-28 21-10-23

### CODE REVIEW

**Review by yaleman — APPROVED** [review]

File: kanidmd_web_ui/src/views/profile.rs [hunk 1]
```diff
@@ -49,7 +49,8 @@ impl TryFrom<Entry> for Profile {
             .cloned()
             .ok_or_else(|| "Missing displayname".to_string())?;
 
-        let groups = entry.attrs.get("memberof").cloned().unwrap_or_default();
+        let mut groups = entry.attrs.get("memberof").cloned().unwrap_or_default();
+        groups.sort();
```
Comment (Firstyear) [inline kanidmd_web_ui/src/views/profile.rs:53]: You should prefer to use sort_unstable https://doc.rust-lang.org/std/vec/struct.Vec.html#method.sort_unstable

**Review by Firstyear — CHANGES_REQUESTED** [review]

File: kanidmd_web_ui/src/views/profile.rs (hunk 1 above)
Comment (czzrr) [reply kanidmd_web_ui/src/views/profile.rs:53]: Makes sense!

**Comment by yaleman** [comment]
Looks like this needs a rebase due to @Firstyear yolo-merging the reorg

**Comment by Firstyear** [comment]
Yep, but it's a 2 line change so not too hard to rebase thankfully. That was why I opted to proceed with the re-org was that the affected PR's weren't super large or hard to rebase.

**Comment by czzrr** [comment]
Rebased. :)

**Review by Firstyear — APPROVED** [review]

**Comment by Firstyear** [comment]
Thanks @czzrr !

**Pull request merged**